src/get-started/codelab.md

```diff
@@ -225,7 +225,12 @@ as well as many other open source packages, on [pub.dev][].
  1. Add `english_words` package to your project as follows:
 
     ```terminal
-    $ flutter pub add english_words
+$ flutter pub add english_words
```

Comment:
I don't believe this de-dent is required.

Reply:
Sir, I corrected it. Please Check it out